Loop.xcodeproj/project.pbxproj

```diff
@@ -1621,19 +1634,21 @@
 			isa = PBXGroup;
 			children = (
 				C1F8B1D122375E4200DD66CF /* BolusProgressTableViewCell.swift */,
-				C1F8B1DB223862D500DD66CF /* BolusProgressTableViewCell.xib */,
 				43B260481ED248FB008CAA77 /* CarbEntryTableViewCell.swift */,
 				4346D1E61C77F5FE00ABAFE3 /* ChartTableViewCell.swift */,
 				431A8C3F1EC6E8AB00823B9C /* CircleMaskView.swift */,
 				43D381611EBD9759007F8C8F /* HeaderValuesTableViewCell.swift */,
 				430D85881F44037000AF2D4F /* HUDViewTableViewCell.swift */,
+				899433B723FE129700FA4BEA /* OverrideBadgeView.swift */,
```

Comment:
(The movement of these preexisting views is just from sorting the folder in Xcode)